src/midiEditorCommands.cpp

```diff
@@ -960,7 +976,9 @@ void cmdMidiMoveToTrack(Command* command) {
 	char* trackName = (char*)GetSetMediaTrackInfo(track, "P_NAME", NULL);
 	if (trackName)
 		s << " " << trackName;
-	s << " item " << itemNum << " " << GetTakeName(take);
+	s << " " << format(
```

Comment:
This one needs a translators comment.
```suggestion
	// Translators: Used when reporting activation of the next/previous track in
	// the MIDI editor. {num} will be replaced with the item number. {name} will
	// be replaced with its name. For example: "item 2 chorus".
	s << " " << format(
```